porcelain/miner_test.go

```diff
@@ -36,13 +37,13 @@ type minerCreate struct {
 }
 
```

Comment:
A bunch of the little changes in this file are to appease the linter deities -- variable/package name conflicts, error messages that are capitalized, etc.